src/jit/lir.h

```diff
@@ -123,39 +141,31 @@ class LIR final
         bool IsSubRange() const;
 
         GenTree* FirstNonPhiNode() const;
+        GenTree* FirstNonPhiOrCatchArgNode() const;
 
```

Comment:
I'm not sure that they're both strictly necessary. I added the latter in order to match an existing function on `BasicBlock`.